packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,204 +24,262 @@ _ref:
+                      - id: paragraph
+                        type: Paragraph
+                        properties:
+                          content: This is a paragraph on Page 1.
+                  - id: input_card
+                    type: Card
+                    blocks:
+                      - id: text_input
+                        type: TextInput
+                        properties:
+                          label:
+                            title: Please Enter Your Name
+                      - id: radio_selector
+                        type: RadioSelector
+                        properties:
+                          label:
+                            title: How Are You Feeling?
+                            colon: false
+                          options:
+                            - label: Meh
+                              value: 1
+                              disabled: false
+                            - label: Okay
+                              value: 2
+                              disabled: false
+                            - label: Good
+                              value: 3
+                              disabled: false
+                            - label: Great
+                              value: 4
+                              disabled: false
+                            - label: Amazing Now That I'm Using Lowdefy
+                              value: 5
+                              disabled: false
+              - id: page2
+                type: PageHeaderMenu
+                properties:
+                  title: Page 2
+                blocks:
+                  - id: content_card
+                    type: Card
+                    blocks:
+                      - id: title
+                        type: Title
+                        properties:
+                          content: Title
+                      - id: paragraph
+                        type: Paragraph
+                        properties:
+                          content: This is a paragraph on Page 2.
+            ```
+
+            In order to keep files neat and generally easier to read and understand, we suggest making use of references and templating.
 
-            ## YAML file extensions
+            ## References and templates
 
-            Both files with the `.yaml` and `.yml` file extensions are supported as YAML files.
+            References are used to split the configuration of an app into logically distinct files, and to reuse configuration in the app. References can be used almost anywhere in the configuration, as long as the configuration remains valid YAML.
 
-            ## JSON instead of YAML
+            Templating can be taken further by referencing [Nunjucks](https://mozilla.github.io/nunjucks/) template files. Nunjucks templates are useful since the template file does not need to be valid yaml before it is hydrated, and features like for-loops and if-statements can be used.
```

Comment:
Templating can be taken further by referencing [Nunjucks](https://mozilla.github.io/nunjucks/) template files by simply adding `.njk to the file extensions for example `.yaml.njk`. Nunjucks templates are useful to hydrate variables at build time, before the yaml is parsed, enabling Nunjucks features like for-loops and if-statements.

Reply:
Change has been made in #1314.